src/domain/challenge/challenge.resolver.fields.ts

```diff
@@ -23,11 +23,7 @@ export class ChallengeResolverFields {
     private challengeService: ChallengeService
   ) {}
 
-  @Roles(
-    RestrictedGroupNames.CommunityAdmins,
-    RestrictedGroupNames.EcoverseAdmins,
-    RestrictedGroupNames.Members
-  )
+  @Roles(RestrictedGroupNames.Members)
```

Comment:
This will restrict only to the Members, not all levels. Roles need to be added explicitly.